App/Redux/NotificationsRedux.ts

```diff
@@ -0,0 +1,33 @@
+import { createAction, ActionType, getType } from 'typesafe-actions'
+import * as TextileTypes from '../Models/TextileTypes'
+
+const actions = {
+  newNotification: createAction('NEW_NOTIFICATION_REQUEST', resolve => {
```

Comment:
Good start I think. I could imagine some other actions we'd want:

1. Acknowledge notification - Fired it the user taps or opens a notification, toggles the status to 'read'
2. Acknowledge all notifications - Fired in a situation like, user opens the notifications screen and is there for at least 5 seconds, and then navigates away from the notifications screen. Marks all unread notifications as read.